libs/async_cuda/src/cuda_future.cpp

```diff
@@ -14,9 +14,10 @@
 #include <hpx/async_cuda/target.hpp>
 #include <hpx/futures/traits/future_access.hpp>
 #include <hpx/modules/errors.hpp>
-#include <hpx/modules/memory.hpp>
```

Comment:
I think this is still needed as this file references the `internal_allocator` exposed by the memory module.

Reply:
it's in 
```
#include <hpx/allocator_support/internal_allocator.hpp>
```
which is included on line 11, so I think it's safe to leave out the memory module.

Reply:
Ok, I didn't notice that. Thanks!